open-sphere-base/mantle/src/main/java/io/opensphere/mantle/infinity/QueryParameters.java

```diff
@@ -121,6 +130,47 @@ public void setBinFieldType(Class<?> binFieldType)
         myBinFieldType = binFieldType;
     }
 
+
+    /**
+     * Gets the bin's width

```

Comment:
End the Javadoc with a period (doccheck complains if it cannot find a complete sentance). Do for all javadocs here.